server/channels/api4/user.go

```diff
@@ -3443,3 +3445,59 @@ func getUsersWithInvalidEmails(c *Context, w http.ResponseWriter, r *http.Reques
 		c.Logger.Warn("Error writing response", mlog.Err(err))
 	}
 }
+
+func getUsersForReporting(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !(c.IsSystemAdmin() && c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionSysconsoleReadUserManagementUsers)) {
+		c.SetPermissionError(model.PermissionSysconsoleReadUserManagementUsers)
+		return
+	}
+
+	sortColumn := "Username"
+	if r.URL.Query().Get("sort_column") != "" {
+		sortColumn = r.URL.Query().Get("sort_column")
+	}
+
+	pageSize := 50
```

Comment:
Very minor nit: I generally come across this as `100` by default, any reason for being `50`?